### DISCUSSION TITLE
Teleprompter placement

### DISCUSSION THREAD

**Discussion by Stopwolf** [OpenMined/courses#199] [Feedback] [answered]
Hi everyone! This is just an idea I had so I thought it might be good to share with the content creators. (Although, I'm not in any way experienced in content creation so I might be totally off here. 😅 )
I couldn't help but notice that from time to time Andrew and Emma were looking down (probably at some notes or something along those lines) while presenting. Maybe you guys could place the notes somewhere in the eye level so it becomes less noticeable for the future viewers. Maybe even switch up sides you place it (sometimes left of the camera, sometimes right), so the presentation looks more natural.
Nonetheless, the course content is great and I'm thoroughly enjoying it! (Didn't have time to finish it tho from all the uni obligations, but I'm close to the end 😄 ) All props to the whole OpenMined crew.. you guys rock! 🤘

**Comment by em-blue** [comment, marked as answer]
This is making me laugh so hard - we actually have a teleprompter giving us the script we wrote, RIGHT in front of the camera lens - we look away to break eye contact *on purpose* otherwise it gets REALLY creepy really quickly when we hold solid eye contact. We *purposefully* look away simply to break eye contact - there's nothing we're looking at off screen 😂! 

Your comment has made my day though - we were very worried it would seem like we made too much eye contact and that we were just reading from a teleprompter in front of the camera lens. In reality, it was actually *very* difficult to look away from the camera (you have to know what line is coming next!), hence why Andrew is able to do it more than me (he's much much more experienced). He also looks down to the side when he's thinking (which is genuine, he always does that). And I think I look to the side or something when thinking - that's all genuine too, it's just who we are as people.

As the listener, you might *think* you want the speaker looking at you the whole time, (which I thought too when we started) but actually it gets really boring/robotic/weird quite quickly. 

Thank you for your comment! I'm glad it doesn't seem like we're reading from a teleprompter in front of the camera ;)

**Reply by Stopwolf** [reply]
Hahaha I'm glad I made your day!
And now that you say that, I actually noticed I do the same thing, I always look to the left somewhere when speaking 😂

**Comment by imflash217** [comment]
This is amazing. 😂
I also felt what @Stopwolf discussed. But, I also felt that the lectures stay in the same position all the time. I was thinking of getting a feel of actual classroom. I am enrolled in my MS degree program here in US and due to Covid-19 all classes have moved online and last semester was tough. One of the major reasons it was tough was that we had to look straight at a very static screen or professor. While in actual classroom movement of professors made a huge impact in shaping the focus and helped the learning process.
@em-blue, I would request that it would be extremely helpful if Andrew and Emma can also move around the space. This would also give us a feel as if the video is from actual classroom instead. 
I loved Andrew's lecture at MIT (https://www.youtube.com/watch?v=4zrU54VIK6k).
Thanks a lot for this amazing course. ⚡️⚡️
